lib/portage/cache/mappings.py

```diff
@@ -439,7 +439,8 @@ def slot_dict_class(keys, prefix="_val_"):
         keys_set = keys
     else:
         keys_set = frozenset(keys)
-    v = _slot_dict_classes.get((keys_set, prefix))
+    cache_key = (keys_set, prefix)
+    v = _slot_dict_classes.get(cache_key)
```

Comment:
Oh, good spot. We don't have any tests for slot_dict_class - we only seem to test portage.cache at all indirectly in `lib/portage/tests/emerge/test_simple.py`.

Any chance of writing up a quick one?

Reply:
There's lib/portage/tests/process/test_pickle.py which tests two different slot_dict_class types: _PackageMetadataWrapperBase (`prefix="_val_"`) and FifoIpcDaemon._files_dict (`prefix=""`). That gives us cases with both types of prefixes.

Reply:
Thanks!